### PR TITLE
Add support for riot ".tag" files

### PR DESCRIPTION
**Pull request by titarenko**

### CODE REVIEW

File: require_node.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def write(index):
 
             region_to_insert = self.view.sel()[0]
 
-            line_is_empty = self.view.lines(region_to_insert)[0].empty()
+            line = self.view.substr(self.view.lines(region_to_insert)[0])
```
Comment (jfromaniello) [inline require_node.py]: What's the purpose of the change in these two lines?

Sorry, I am not very familiar with python and the sublime api

Comment (titarenko) [inline require_node.py]: that's to check whether line has withespaces (and actually is empty - try to insert require statement in function body - in such case there will be indentation). empty() method returns false in such cases (spaces, tabs)

Comment (jfromaniello) [inline require_node.py]: excellent!

**Comment by jfromaniello** [comment]
Thanks and sorry for the delay. I just added a `2.1.0` tag on it.
